### PR TITLE
gnrc_sixlowpan_frag_rb: fix doc on DO_NOT_OVERRIDE

### PR DESCRIPTION
**Pull request by miri64**
### Contribution description
While working on https://github.com/RIOT-OS/RIOT/pull/13244 I re-read the doc on the corresponding option in 6LoWPAN that this PR introduces, to have the two unified, and found it to be a bit misleading (even though we discussed the documentation at length in https://github.com/RIOT-OS/RIOT/pull/13129). This PR fixes the documentation to be a bit more clearer.

### Testing procedure
Read.

### Issues/PRs references
None

### CODE REVIEW

File: sys/net/gnrc/network_layer/sixlowpan/frag/rb/Kconfig [hunk 1]
```diff
@@ -21,7 +21,7 @@ config GNRC_SIXLOWPAN_FRAG_RBUF_TIMEOUT_US
     default 3000000
 
 config GNRC_SIXLOWPAN_FRAG_RBUF_DO_NOT_OVERRIDE
-    bool "Keep all but oldest fragment when reassembly buffer is full"
+    bool "Do not override oldest datagram when reassembly buffer is full on incoming fragment"
```
Comment (leandrolanzieri) [inline sys/net/gnrc/network_layer/sixlowpan/frag/rb/Kconfig]: Could this be shortened to 79 characters? Extra info can be included under `help`

Comment (miri64) [reply sys/net/gnrc/network_layer/sixlowpan/frag/rb/Kconfig]: Can I just break lines here or how do I keep the doc in line with the `@brief`?

Comment (miri64) [reply sys/net/gnrc/network_layer/sixlowpan/frag/rb/Kconfig]: Done.

**Comment by miri64** [comment]
Squashed!

**Pull request merged**